code/game/gamemodes/endgame/xmas/krampus.dm

```diff
@@ -28,12 +28,60 @@
 	real_name = "Krampus"
 	status_flags = GODMODE|CANPUSH
 
-	New(var/new_loc)
-		h_style = "Bald"
-		..(new_loc, "Krampus")
-		maxHealth=999999
-		health=999999
+/mob/living/carbon/human/krampus/New(var/new_loc)
+	h_style = "Bald"
+	..(new_loc, "Krampus")
+	maxHealth=999999
+	health=999999
+	var/obj/item/weapon/krampus/sack = new /obj/item/weapon/krampus(src)
+	put_in_hands(sack)
 
+/mob/living/carbon/human/krampus/proc/SackEm(var/mob/M)
+	if(!istype(M))
+		return
+
+	var/datum/admins/Krampus = client.holder
+	if(!(Krampus && check_rights(R_BAN)))
+		return
+
+	var/response = alert("Ban them too, or just sack them?",,"Ban", "Sack", "Cancel")
+	if(response == "Cancel")
+		return
+	if(response == "Ban" && !M.ckey)
+		to_chat(src, "You can only do that to mobs with a ckey.")
+		return
+
+	M.drop_all()
+	M.loc = src //need somewhere to store them while they're getting banned
+	var/g
```

Comment:
If you say var/g = "thing" you can save TWO HOLE LINES as you won't need that extra else statement